#24373 : Documentation/Navbar : Refer to utility classes used. Hidden when printing.

aavmurphy opened the pull request:
refer to the  (not navbar) classes used for newbies
add: navbars don't print
give the 1st example a drop-down (as its the template people will copy and adapt)

aavmurphy committed:
Refer to utility classes uses. Add Printing section

refer to (not navbar) classes used for newbies
navbars don't print, so add example of a print-only alternative

aavmurphy committed:
typos, add links

typo in {% example %}
add links to component pages

XhmikosR commented on docs/4.0/components/navbar.md:
```diff
@@ -29,7 +29,10 @@ Navbars come with built-in support for a handful of sub-components. Choose from
-Here's an example of all the sub-components included in a responsive light-themed navbar that automatically collapses at the `lg` (large) breakpoint.
+Here's an example of all the sub-components.
+- the navbar is light-themed
+- its responsive - it automatically collapses at the `lg` (large) breakpoint.
```
`It's`

aavmurphy committed:
grammar

aavmurphy committed:
Add a dropdown with divider to the main example

aavmurphy committed:
change id name to stop travis error

I assume id="camelCase" is the standard for id names, rather than the clearly superior id="underscore_words"

mdo reviewed (changes requested):
Rather than add an entire new section to this page, let's put this under the `How it works` list.

aavmurphy committed:
Merge branch 'v4-dev' into v4-dev

aavmurphy committed:
review changes

still like to emphasise the foreign utility classes in the main example
bg-light's name is self explanatory (so its obvious where to look it up), but mr-sm-2's name isn't at first

aavmurphy committed:
review changes

aavmurphy committed:
grammar

aavmurphy committed:
.fixed-top/bottom don't reserve space on the screen

... as they are position:fixed. 
better wording welcome

aavmurphy commented:
@mdo please re-check. have added 1 more thing (fixed-top/bottom uses position:fixed), so doesn't reserve a gap on the screen. couldn't thing of a nice way to phrase it though. thanks

mdo committed:
copyediting

- rewrite utils line and link to the utils pages
- restore the paragraphs to the placement section
- fix formatting and grammar

mdo reviewed (approved):
I pushed another commit to fine tune some things here around the language and formatting of the content—more links to utilities pages, better formatting of paragraphs and lists, etc.

Thanks for the back and forth!

aavmurphy commented:
Thanks!
